lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -191,25 +196,31 @@ def add_payment(post, payment, options)
       end
 
       def add_network_tokenization_card(post, payment, options)
-        post[:processingInformation][:paymentSolution] = PAYMENT_SOLUTION[payment.source]
         post[:processingInformation][:commerceIndicator] = 'internet' unless card_brand(payment) == 'jcb'
 
         post[:paymentInformation][:tokenizedCard] = {
           number: payment.number,
           expirationMonth: payment.month,
           expirationYear: payment.year,
           cryptogram: payment.payment_cryptogram,
-          transactionType: '1',
```

Comment:
Why not just keep it here an do something like payment_method.source == :network_token ? '3' : '1'?

Reply:
We could do that!